.github/workflows/ci.yml

```diff
@@ -39,8 +39,8 @@ jobs:
     strategy:
       matrix:
         platforms: [
-          'iOS_13,tvOS_13,watchOS_6',
-          'macOS_10_15',
+          'iOS_13,watchOS_6,macOS_10_15',
+          'tvOS_13',
```

Comment:
Can you help me understand the changes here and below? Why do we need to move things around?

Reply:
We don't _need_ to move these things around, but I'm playing with moving them around to see if I can lower the overall wall time taken to complete a run. Some jobs complete faster than others, and I'm trying to optimize such that the sharded jobs have equal-ish runtimes. I may play with this setup once more before merging – still gathering data.